python/lsst/ts/externalscripts/auxtel/latiss_intra_extra_focal_data.py

```diff
@@ -0,0 +1,172 @@
+            remotes=remotes,
+            descr="Procedure for taking intra and extra focal data of the Rubin "
+            "Auxiliary Telescope with LATISS using the specified degrees of freedom "
+            "on the targets of interest.",
+        )
+
+        self.log.info(
+            "LATISS Wavefront Estimation Pipeline initialized. Perform optical "
+            "Take intra and extra focal data with the Rubin Auxiliary Telescope with LATISS "
+            "applying the specified degrees of freedom."
+        )
+
+    @classmethod
+    def get_schema(cls):
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_externalscripts/blob/main/python/lsst/ts/externalscripts/"
+            "auxtel/latiss_intra_extra_focal_data.py"
+        )
+        schema = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: LatissIntraExtraFocalData v1
+        description: Configuration for making a LATISS calibrations SAL Script.
+        type: object
+        properties:
+            offset_x:
+                description: x degree of freedom offset in mm.
+                type: number
+                default: 0.0
+              offset_y:
+                description: y degree of freedom offset in mm.
+                type: number
+                default: 0.0
+              offset_z:
+                description: z degree of freedom offset in mm.
+                type: number
+                default: 0.0
+              offset_Rx:
+                description: Rx degree of freedom offset in degrees.
+                type: number
+                default: 0.0
+              offset_Ry:
+                description: Ry degree of freedom offset in degrees.
+                type: number
+                default: 0.0
+              offset_m1:
+                description: M1 pressure degree of freedom offset in Pa.
+                type: number
+                default: 0.0
+                maximum: 0.0
+        additionalProperties: false
+        """
+        schema_dict = yaml.safe_load(schema)
+        base_schema_dict = super(LatissIntraExtraFocalData, cls).get_schema()
+
+        for prop in base_schema_dict["properties"]:
+            schema_dict["properties"][prop] = base_schema_dict["properties"][prop]
+
+        return schema_dict
+
+    async def additional_configuration(self, config: types.SimpleNamespace) -> None:
+        """Additional configuration.
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+        # set dofs
+        self.offset_x = config.offset_x
+        self.offset_y = config.offset_y
+        self.offset_z = config.offset_z
+        self.offset_Rx = config.offset_Rx
+        self.offset_Ry = config.offset_Ry
+        self.offset_m1 = config.offset_m1
+
+    async def arun(self, checkpoint: bool = False):
+        """Takes Intr and Extra Focal Data.
```

Comment:
```
Run script operation encapsulating the checkpoints to allow running standalone.
```

If you want to describe what the method does, maybe add a "Notes" session. But I am not sure this is necessary.